src/js/targeted-content.js

```diff
@@ -1,9 +1,15 @@
 const initTargetedContent = () => {
     try {
-        const content = document.getElementsByClassName(
+        const pageContents = document.getElementsByClassName(
+            'cads-main-content'
+        );
+        const pageContent =
+            pageContents && pageContents.length ? pageContents[0] : document;
```

Comment:
Did we include a build system in DS? If so, then this line could be replaced by `const pageContent = pageContents?.[0] : document`. But need to make sure this will be polyfilled for earlier browsers.

Reply:
Done